docs/source/outputs.rst

```diff
@@ -0,0 +1,148 @@
+.. _outputs:
+
+Eureka! Outputs
+===============
+
+Stage 2 through Stage 5 of ``Eureka!`` can be configured to output plots of the pipeline's interim results as well as the data required to run further stages. 
+
+
+.. _s2-out:
+
+Stage 2 Outputs
+---------------
+
+If ``skip_extract_1d`` is set in the Stage 2 ECF, the 1-dimensional spectrum will not be extracted, and no plots will be made. Otherwise, Stage 2 will extract the 1-dimensional spectrum from the calibrated images, and will plot the spectrum.
+
+.. figure:: ../media/S2_out/fig2100.png
+   :alt: Stage 2 1-dimensional spectrum plot
+
+   Stage 2 output: 1-Dimensional Spectrum Plot
+
+
+.. _s3-out:
+
+Stage 3 Outputs
+---------------
+
+In Stage 3 through Stage 5, output plots are controlled with the ``isplots_SX`` parameter. The resulting plots are cumulative: setting ``isplots_S3 = 5`` will also create the plots specified in ``isplots_S3 = 3`` and ``isplots_S3 = 1``.
+
+In Stage 3:
+   - If ``isplots_S3`` = 1: ``Eureka!`` will plot the 2-dimensional, non-drift-corrected light curve. 
+
+   .. figure:: ../media/S3_out/fig3101.png
+      :alt: Stage 3 2-dimensional spectrum plot
+
+      Stage 3 output: 2-Dimensional Spectrum Plot
+
+   - If ``isplots_S3`` = 3: ``Eureka!`` will plot the results of the background and optimal spectral extraction steps for each exposure in the observation, as well as the source position on the detector.
+
+   .. figure:: ../media/S3_out/fig3301.png
+      :alt: Stage 3 background subtracted flux plot
+
+      Stage 3 output: Background Subtracted Flux Plot
+
+   .. figure:: ../media/S3_out/fig3302.png
+      :alt: Stage 3 1-dimensional spectrum plot
+
+      Stage 3 output: 1-Dimensional Spectrum Plot
+
+   .. figure:: ../media/S3_out/fig3303.png
+      :alt: Stage 3 source position fit
+
+      Stage 3 output: Source Position Fit Plot
+
+   .. figure:: ../media/S3_out/fig3305.png
+      :alt: Stage 3 weighting profile
+
+      Stage 3 output: Weighting Profile Plot
+
+   - If ``isplots_S3`` = 5: ``Eureka!`` will plot the Subdata plots from the optimal spectral extraction step.
+
+   .. figure:: ../media/S3_out/fig3501.png
+      :alt: Stage 3 subdata plot
+
+      Stage 3 output: Spectral Extraction Subdata Plot
+   
+.. _s4-out:
+
+Stage 4 Outputs
+---------------
+
+In Stage 4:
+   - If ``isplots_S4`` = 1: ``Eureka!`` will plot the spectral drift per exposure, and the drift-corrected 2-dimensional lightcurve with extracted bins overlaid.
+
+   .. figure:: ../media/S4_out/fig4101.png
+      :alt: Stage 4 spectral drift plot
+
+      Stage 4 output: Spectral Drift Plot
+
+   .. figure:: ../media/S4_out/fig4201.png
```

Comment:
Are you able to get a newer copy of this figure? The formatting has changed since you made this one

Reply:
Once that's edited though, I think everything looks great!